src/pages/home/ReportScreen.js

```diff
@@ -400,6 +401,14 @@ function ReportScreen({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    useEffect(() => {
+        if (!isHydrated) {
+            return;
+        }
+
+        fetchReportIfNeeded();
+    }, [isHydrated]);
```

Comment:
This change was added to solve an issue when creating a new workspace room, Chat Report is shown as the room name.

https://github.com/Expensify/App/assets/50919443/bf145cad-6817-45f2-aba9-34c201cd2f83

That's because the data is not ready yet here and the report name becomes Chat Report.
https://github.com/Expensify/App/blob/main/src/libs/actions/Report.ts#L552-L556